### ISSUE TITLE
Share Feedback button crashes the editor

### ISSUE THREAD

**Issue by MohamedElashri** [zed-industries/zed#6430]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

With the latest zed versions `0.119.17` and `0.119.18` clicking on share feedback button will cause a zed editor to crash every time this is tried.  

### Environment

Ironically, the same thing happens when trying to run `﻿copy system specs into clipboard` it will cause crashes. But I think this information might help from crash report. 

```
Model: MacBookPro17,1, BootROM 10151.81.1, proc 8:4:4 processors, 16 GB, SMC 
Graphics: Apple M1, Apple M1, Built-In
Display: M215BH, 1920 x 1080 (1080p FHD - Full High Definition), Main, MirrorOff, Online
Display: LCD TV, 1920 x 1080 (1080p FHD - Full High Definition), MirrorOff, Online
Memory Module: LPDDR4, Micron
Process:               zed [26946]
Path:                  /Applications/Zed.app/Contents/MacOS/zed
Identifier:            dev.zed.Zed
Version:               0.119.18 (20240124.192413)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-01-24 14:52:26.7939 -0500
OS Version:            macOS 14.3 (23D56)
Report Version:        12
Anonymous UUID:        <redacted>

Time Awake Since Boot: 28000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  main  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x000000000000012c
Exception Codes:       0x0000000000000001, 0x000000000000012c

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [26946]

VM Region Info: 0x12c is not in any region.  Bytes before following region: 68719476436
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      commpage (reserved)        1000000000-7000000000   [384.0G] ---/--- SM=NUL  ...(unallocated)

```

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

```
2024-01-24T18:19:57 [INFO] ========== starting zed ==========
2024-01-24T18:19:57 [INFO] Opening main db
2024-01-24T18:19:58 [INFO] Opening main db
2024-01-24T18:19:59 [INFO] Node runtime install_if_needed
2024-01-24T18:20:06 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T18:20:06.028Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T18:20:06.028Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T18:20:06 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "56a6b5d9-9ad3-420a-9bb0-c7a4462cfcdf",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T18:20:06 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T18:20:06.874Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T18:20:06.874Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T18:20:17 [INFO] set environment variables from shell:/bin/zsh, path:/opt/homebrew/Caskroom/miniforge/base/bin:/opt/homebrew/Caskroom/miniforge/base/condabin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.1/shims:/Users/melashri/.pyenv/shims:/Users/melashri/.nvm/versions/node/v16.13.1/bin:/opt/homebrew/opt/python@3.8/bin:/usr/local/bin:/opt/homebrew/lib/ruby/gems/3.3.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/opt/python@3.11/bin:/usr/local/Cellar/python@3.10/3.10/bin:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mohamedelashri/Applications/quarto/bin:/Users/melashri/.nix-profile/bin:/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin:/usr/local/bin:/usr/bin:/usr/sbin:/bin:/sbin:/Users/melashri/.fig/bin:/Users/melashri/.local/bin:/Users/mohamedelashri/Library/Application Support/JetBrains/Toolbox/scripts:/Applications/ShellHistory.app/Contents/Helpers:/opt/homebrew/opt/fzf/bin:/Applications/ShellHistory.app/Contents/Helpers:/Users/melashri/.local/bin
2024-01-24T18:20:18 [INFO] downloaded update. path:"/var/folders/kt/_8dpvtks60v1dwnmpsxpzg4h0000gn/T/zed-auto-update.URZSswHqkz6h/Zed.dmg"
2024-01-24T18:26:25 [ERROR] oneshot canceled
2024-01-24T18:26:38 [INFO] ========== starting zed ==========
2024-01-24T18:26:38 [INFO] Opening main db
2024-01-24T18:26:39 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T18:26:39 [INFO] Opening main db
2024-01-24T18:26:39 [INFO] Node runtime install_if_needed
2024-01-24T18:26:42 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T18:26:42.564Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T18:26:42.564Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T18:26:42 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "570dbbb9-18de-4f84-8200-ac9284c579db",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T18:26:43 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T18:26:43.241Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T18:26:43.241Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T18:26:54 [INFO] set environment variables from shell:/bin/zsh, path:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.1/shims:/Users/melashri/.pyenv/shims:/Users/melashri/.nvm/versions/node/v16.13.1/bin:/opt/homebrew/opt/python@3.8/bin:/usr/local/bin:/opt/homebrew/lib/ruby/gems/3.3.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/opt/python@3.11/bin:/usr/local/Cellar/python@3.10/3.10/bin:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mohamedelashri/Applications/quarto/bin:/opt/homebrew/Caskroom/miniforge/base/bin:/opt/homebrew/Caskroom/miniforge/base/condabin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.1/shims:/Users/melashri/.nvm/versions/node/v16.13.1/bin:/opt/homebrew/opt/python@3.8/bin:/usr/local/bin:/opt/homebrew/lib/ruby/gems/3.3.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/opt/python@3.11/bin:/usr/local/Cellar/python@3.10/3.10/bin:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mohamedelashri/Applications/quarto/bin:/Users/melashri/.nix-profile/bin:/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin:/usr/local/bin:/usr/bin:/usr/sbin:/bin:/sbin:/Users/melashri/.fig/bin:/Users/melashri/.local/bin:/Users/mohamedelashri/Library/Application Support/JetBrains/Toolbox/scripts:/Applications/ShellHistory.app/Contents/Helpers:/opt/homebrew/opt/fzf/bin:/Applications/ShellHistory.app/Contents/Helpers:/Users/melashri/.local/bin:/Users/mohamedelashri/Library/Application Support/JetBrains/Toolbox/scripts:/Applications/ShellHistory.app/Contents/Helpers:/Applications/ShellHistory.app/Contents/Helpers:/Users/melashri/.local/bin
2024-01-24T18:40:44 [INFO] open paths ["/Users/melashri/.config/zed/settings.json"]
2024-01-24T18:40:44 [INFO] Opening main db
2024-01-24T18:40:44 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:40:44 [INFO] starting language server "json-language-server", path: "/Users/melashri/.config/zed", id: 1
2024-01-24T18:40:45 [INFO] Node runtime install_if_needed
2024-01-24T18:40:48 [INFO] Node runtime install_if_needed
2024-01-24T18:40:52 [INFO] Node runtime install_if_needed
2024-01-24T18:52:09 [ERROR] crates/lsp/src/lsp.rs:615: oneshot canceled
2024-01-24T18:52:09 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:52:57 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:05 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:05 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:05 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:08 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:09 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:09 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:09 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:13 [INFO] open paths ["/Users/melashri/.config/zed/settings.json"]
2024-01-24T18:53:13 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:13 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:13 [INFO] starting language server "json-language-server", path: "/Users/melashri/.config/zed/settings.json", id: 2
2024-01-24T18:53:17 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:17 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:21 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:21 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:22 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:22 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:24 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:24 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:33 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:33 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:40 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:40 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:42 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:42 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:43 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:43 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:45 [ERROR] crates/project/src/project.rs:4570: Unhandled method workspace/symbol
2024-01-24T18:53:48 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:53:48 [INFO] Initializing default prettier with plugins {}
2024-01-24T18:54:03 [ERROR] crates/lsp/src/lsp.rs:615: oneshot canceled
2024-01-24T18:54:03 [INFO] Initializing default prettier with plugins {}
2024-01-24T19:50:51 [INFO] set status on client 0: Authenticating
2024-01-24T19:50:57 [INFO] set status on client 46139: Connecting
2024-01-24T19:50:58 [INFO] response dropped without fully consuming the response body, connection won't be reused
Aborting a response without fully consuming the response body can result in sub-optimal performance. See https://github.com/sagebind/isahc/wiki/Connection-Reuse#closing-connections-early.
2024-01-24T19:50:59 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:50:59 [INFO] add connection to peer
2024-01-24T19:50:59 [INFO] waiting for server hello
2024-01-24T19:50:59 [INFO] got server hello
2024-01-24T19:50:59 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57471 })
2024-01-24T19:50:59 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57471 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:51:16 [ERROR] crates/ai/src/providers/open_ai/completion.rs:232: NotPresent
2024-01-24T19:51:16 [ERROR] crates/ai/src/providers/open_ai/completion.rs:232: NotPresent
2024-01-24T19:51:31 [INFO] Opening main db
2024-01-24T19:51:42 [INFO] ========== starting zed ==========
2024-01-24T19:51:42 [INFO] Opening main db
2024-01-24T19:51:43 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T19:51:43 [INFO] Opening main db
2024-01-24T19:51:43 [INFO] set status on client 0: Authenticating
2024-01-24T19:51:43 [INFO] set status on client 46139: Connecting
2024-01-24T19:51:43 [INFO] Opening main db
2024-01-24T19:51:43 [INFO] Node runtime install_if_needed
2024-01-24T19:51:44 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:51:44 [INFO] add connection to peer
2024-01-24T19:51:44 [INFO] waiting for server hello
2024-01-24T19:51:44 [INFO] got server hello
2024-01-24T19:51:44 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57505 })
2024-01-24T19:51:44 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57505 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:51:44 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:51:44.924Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:51:44.924Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T19:51:44 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "03c20f35-e6c2-4b08-a22b-c3fb6ab77354",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T19:51:45 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:51:45.600Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:51:45.600Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T19:51:49 [INFO] set environment variables from shell:/bin/zsh, path:/opt/homebrew/Caskroom/miniforge/base/bin:/opt/homebrew/Caskroom/miniforge/base/condabin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.1/shims:/Users/melashri/.pyenv/shims:/Users/melashri/.nvm/versions/node/v16.13.1/bin:/opt/homebrew/opt/python@3.8/bin:/usr/local/bin:/opt/homebrew/lib/ruby/gems/3.3.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/opt/python@3.11/bin:/usr/local/Cellar/python@3.10/3.10/bin:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mohamedelashri/Applications/quarto/bin:/Users/melashri/.nix-profile/bin:/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin:/usr/local/bin:/usr/bin:/usr/sbin:/bin:/sbin:/Users/melashri/.fig/bin:/Users/melashri/.local/bin:/Users/mohamedelashri/Library/Application Support/JetBrains/Toolbox/scripts:/Applications/ShellHistory.app/Contents/Helpers:/opt/homebrew/opt/fzf/bin:/Applications/ShellHistory.app/Contents/Helpers:/Users/melashri/.local/bin
2024-01-24T19:51:57 [INFO] downloaded update. path:"/var/folders/kt/_8dpvtks60v1dwnmpsxpzg4h0000gn/T/zed-auto-update.wFX70MUqN1H8/Zed.dmg"
2024-01-24T19:53:56 [INFO] ========== starting zed ==========
2024-01-24T19:53:56 [INFO] Opening main db
2024-01-24T19:53:56 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T19:53:56 [INFO] Opening main db
2024-01-24T19:53:56 [INFO] set status on client 0: Authenticating
2024-01-24T19:53:56 [INFO] set status on client 46139: Connecting
2024-01-24T19:53:56 [INFO] Opening main db
2024-01-24T19:53:57 [INFO] Node runtime install_if_needed
2024-01-24T19:53:58 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:53:58 [INFO] add connection to peer
2024-01-24T19:53:58 [INFO] waiting for server hello
2024-01-24T19:53:58 [INFO] got server hello
2024-01-24T19:53:58 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57648 })
2024-01-24T19:53:58 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57648 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:53:58 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:53:58.454Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:53:58.454Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T19:53:58 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "913640ab-9616-4f62-80b8-efc75ac9621c",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T19:53:59 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:53:59.108Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:53:59.108Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T19:54:47 [INFO] ========== starting zed ==========
2024-01-24T19:54:47 [INFO] Opening main db
2024-01-24T19:54:47 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T19:54:47 [INFO] Opening main db
2024-01-24T19:54:47 [INFO] set status on client 0: Authenticating
2024-01-24T19:54:47 [INFO] set status on client 46139: Connecting
2024-01-24T19:54:47 [INFO] Opening main db
2024-01-24T19:54:47 [INFO] Node runtime install_if_needed
2024-01-24T19:54:48 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:54:48 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:54:48.919Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:54:48.919Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T19:54:48 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "7f786478-365d-459e-971f-986208740fb1",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T19:54:49 [INFO] add connection to peer
2024-01-24T19:54:49 [INFO] waiting for server hello
2024-01-24T19:54:49 [INFO] got server hello
2024-01-24T19:54:49 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57698 })
2024-01-24T19:54:49 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57698 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:54:49 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:54:49.490Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:54:49.490Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T19:54:53 [INFO] set environment variables from shell:/bin/zsh, path:/opt/homebrew/Caskroom/miniforge/base/bin:/opt/homebrew/Caskroom/miniforge/base/condabin:/opt/homebrew/Cellar/pyenv-virtualenv/1.2.1/shims:/Users/melashri/.pyenv/shims:/Users/melashri/.nvm/versions/node/v16.13.1/bin:/opt/homebrew/opt/python@3.8/bin:/usr/local/bin:/opt/homebrew/lib/ruby/gems/3.3.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/opt/python@3.11/bin:/usr/local/Cellar/python@3.10/3.10/bin:/opt/local/bin:/opt/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mohamedelashri/Applications/quarto/bin:/Users/melashri/.nix-profile/bin:/run/current-system/sw/bin:/nix/var/nix/profiles/default/bin:/usr/local/bin:/usr/bin:/usr/sbin:/bin:/sbin:/Users/melashri/.fig/bin:/Users/melashri/.local/bin:/Users/mohamedelashri/Library/Application Support/JetBrains/Toolbox/scripts:/Applications/ShellHistory.app/Contents/Helpers:/opt/homebrew/opt/fzf/bin:/Applications/ShellHistory.app/Contents/Helpers:/Users/melashri/.local/bin
2024-01-24T19:57:13 [INFO] ========== starting zed ==========
2024-01-24T19:57:13 [INFO] Opening main db
2024-01-24T19:57:13 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T19:57:13 [INFO] Opening main db
2024-01-24T19:57:14 [INFO] set status on client 0: Authenticating
2024-01-24T19:57:14 [INFO] set status on client 46139: Connecting
2024-01-24T19:57:14 [INFO] Opening main db
2024-01-24T19:57:14 [INFO] Node runtime install_if_needed
2024-01-24T19:57:15 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:57:15 [INFO] add connection to peer
2024-01-24T19:57:15 [INFO] waiting for server hello
2024-01-24T19:57:15 [INFO] got server hello
2024-01-24T19:57:15 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57826 })
2024-01-24T19:57:15 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57826 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:57:15 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:57:15.588Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:57:15.588Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T19:57:15 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "8aa6fbf7-e2c8-4e93-9780-1e6700d6fbb5",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T19:57:16 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:57:16.146Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:57:16.146Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
2024-01-24T19:58:17 [INFO] ========== starting zed ==========
2024-01-24T19:58:17 [INFO] Opening main db
2024-01-24T19:58:17 [ERROR] crates/zed/src/zed.rs:538: EOF while parsing a value at line 1 column 0
2024-01-24T19:58:17 [INFO] Opening main db
2024-01-24T19:58:17 [INFO] set status on client 0: Authenticating
2024-01-24T19:58:18 [INFO] set status on client 46139: Connecting
2024-01-24T19:58:18 [INFO] Opening main db
2024-01-24T19:58:18 [INFO] Node runtime install_if_needed
2024-01-24T19:58:18 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-01-24T19:58:19 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:58:19.401Z] Agent service starting",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:58:19.401Z]",
  "extra": [
    "Agent service starting"
  ]
}
2024-01-24T19:58:19 [INFO] 0 unhandled notification client/registerCapability:
{
  "registrations": [
    {
      "id": "0829dab5-0035-447b-86e6-d92350ebc66f",
      "method": "workspace/didChangeWorkspaceFolders",
      "registerOptions": {}
    }
  ]
}
2024-01-24T19:58:19 [INFO] add connection to peer
2024-01-24T19:58:19 [INFO] waiting for server hello
2024-01-24T19:58:19 [INFO] got server hello
2024-01-24T19:58:19 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 274, id: 57888 })
2024-01-24T19:58:19 [INFO] set status on client 46139: Connected { peer_id: PeerId { owner_id: 274, id: 57888 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-01-24T19:58:19 [INFO] 0 unhandled notification LogMessage:
{
  "level": 0,
  "message": "[DEBUG] [agent] [2024-01-24T19:58:19.928Z] Telemetry initialized",
  "metadataStr": "[DEBUG] [agent] [2024-01-24T19:58:19.928Z]",
  "extra": [
    "Telemetry initialized"
  ]
}
```

**Comment by maxdeviant** [comment]
Is this fixed by #6431?

**Comment by MohamedElashri** [comment]
> Is this fixed by #6431?

Yes the new release fixed it. Thanks
